test/lib/pgBackRestTest/Module/Real/RealAllTest.pm

```diff
@@ -506,7 +508,7 @@ sub run
             # a read error instead of an open error
```

Comment:
"which is a read error instead of an open error"  comment contradicts the test since the test expects an OPEN error.

Reply:
Fixed comment.  Talk about attention to detail!